src/index.ts

```diff
@@ -57,29 +76,55 @@ function assertIOS(name: string) {
   return isIOS
 }
 
+const mapStateToNativeState = (state: LiveActivityState): NativeLiveActivityState => {
```

Comment:
This would not be necessary of we change parameter to `progressBar` in the native module.